__tests__/index.js

```diff
@@ -30,14 +30,15 @@ async function execCommand(
   }
 
   return new Promise((resolve, reject) => {
+    const {...cleanedEnv} = {...process.env};
+    cleanedEnv['YARN_SILENT'] = 0;
+    delete cleanedEnv['FORCE_COLOR'];
```

Comment:
Isn't it safe to force this value to be `0` as well?

Reply:
Unfortunately, setting FORCE_COLOR to 0 is not enough. All of the packages just checks if this variable is defined. Here is an example from `supports-color` library (which is a dependency of yarn):

```javascript
if ('FORCE_COLOR' in process.env) {
	return true;
}
```